l10n/de_DE.js

```diff
@@ -383,6 +383,9 @@ OC.L10N.register(
     "Create data" : "Daten erstellen",
     "Update data" : "Daten aktualisieren",
     "Delete data" : "Daten löschen",
+    "Internal Link" : "Interner Link",
```

Comment:
Forgot to mention, the commit with translations can be dropped. We sync those separately to [transifex](https://explore.transifex.com/nextcloud/nextcloud/) and pull updated translations from there nightly, so they would be dropped on the next sync until they are translated over there.